src/content/docs/it/recipes.mdx

```diff
@@ -1,37 +1,20 @@
-
-- [Importare immagini utilizzando un nome file dinamico](https://vaihe.com/blog/astro/astro-dynamic-image-prop/)
-- [Aggiungere transizioni di pagina animate con Swup](https://navillus.dev/blog/astro-plus-swup)
-- [Utilizzare UnoCSS in Astro](https://www.elian.codes/blog/23-02-11-implementing-unocss-in-astro/)
-- [Costruire una tabella dei contenuti utilizzando le intestazioni dei file Markdown di Astro](https://kld.dev/building-table-of-contents/)
-- [Creare un plugin Remark per rimuovere i "righini" dai tuoi file Markdown](https://eatmon.co/blog/remove-runts-markdown/)
-- [Aggiungere al tuo sito la funzionalità di ricerca con Pagefind](https://blog.otterlord.dev/post/astro-search/)
-- [Configurare VSCode, ESLint e Prettier in progetti Astro](https://patheticgeek.dev/blog/astro-prettier-eslint-vscode)
-- [Ottimizzare le build di progetti Astro: modificare i file HTML post-build](https://straffesites.com/en/blog/optimize-astro-html-post-build)
-- [Aggiungere la modalità scura in Astro con Tailwind CSS](https://www.kevinzunigacuellar.com/blog/dark-mode-in-astro/)
-- [Generare sitemaps localizzate nei tuoi progetti Astro con Storyblok](https://straffesites.com/en/blog/localized-sitemap-astro-storyblok)
-- [Integrare Prettier con Astro e Tailwind CSS](https://straffesites.com/en/blog/integrate-prettier-astro-tailwindcss)
-- [Aggiungere un feed RSS ad Astro utilizzando le storie di Storyblok](https://straffesites.com/en/blog/rss-feed-astro-storyblok)
-- [Utilizzare tRPC in Astro](https://www.thomasledoux.be/blog/using-trpc-astro-islands-react)
-- [Come rendere multilingue il tuo sito Astro con Crowdin: una guida alla localizzazione in Astro](https://crowdin.com/blog/2023/06/21/astro-localization-and-i18n)
-- [Aggiungere le immagini di un blog post al feed RSS Astro](https://webreaper.dev/posts/astro-rss-feed-blog-post-images/)
-- [Adottare un'architettura micro-frontends con Astro](https://medium.com/@sergio.a.soria/setting-up-micro-frontends-with-astro-and-ecma-script-modules-137340d2c520)
-- [Aggiungere autenticazione basata su nome utente e password con Lucia](https://lucia-auth.com/guidebook/sign-in-with-username-and-password/astro)
-- [Aggiungere Github OAuth con Lucia](https://lucia-auth.com/guidebook/github-oauth/astro)
-- [Integrare Sentry con Astro](https://akashrajpurohit.com/blog/seamless-error-tracking-integrating-sentry-with-astro/)
-- [Configura le bozze delle pagine in modo efficace in Astro con la creazione di contenuti basata sulla configurazione](https://akashrajpurohit.com/blog/set-up-draft-pages-effectively-in-astro-with-configdriven-content-authoring/)
+## Risorse della comunità
+
+import ReadMore from '~/components/ReadMore.astro'
+
+<ReadMore>Guarda ancora più ricette scritte e mantenute dalla community nelle nostre [ricette e guide della community](/it/community-resources/content/#ricette-e-guide)</ReadMore>
```

Comment:
```suggestion
<ReadMore>Guarda ancora più ricette scritte e mantenute dalla community nelle nostre [ricette e guide della community](/it/community-resources/content/#recipes-and-guides)</ReadMore>
```

Reply:
As I said in [this comment](https://github.com/withastro/docs/pull/6835#issuecomment-1934819625) the broken link will be resolved once the mentioned PR I made is merged. The changes are separated because they were in a different folder and I created 3 different PRs to keep everything organized instead of having a big PR. If I committed this change the link would be broken in the main branch. Hope this clears everything :)

Reply:
Got it, thanks for breaking that down for me!